### PR TITLE
Make highlight commands ephemeral

### PR DESCRIPTION
**Pull request by CircuitSacul**

### CODE REVIEW

**Comment by MithicSpirit** [comment]
Should the non-error response to `/highlights list` also be ephemeral?

**Comment by CircuitSacul** [comment]
Yes, I missed that one.

File: mcodingbot/plugins/highlights.py [hunk 1]
```diff
@@ -103,19 +107,20 @@ async def list(ctx: Context) -> None:
...
     if not highlights:
-        await ctx.respond("You do not have any highlights.")
+        await ctx.respond("You do not have any highlights.", ephemeral=True)
         return
 
     embed = hikari.Embed(
         title="Your Highlights",
         description="\n".join(hl.highlight for hl in highlights),
         color=CONFIG.theme,
+        ephemeral=True,
```
Comment (MithicSpirit) [inline mcodingbot/plugins/highlights.py]: Shouldn't `ephemeral` be in `ctx.respond` instead?

Comment (MithicSpirit) [reply mcodingbot/plugins/highlights.py]: Fixed right as I commented lmao

**Review by MithicSpirit — APPROVED** [review]
lgtm

**Review by trag1c — APPROVED** [review]

**Review by Lunarmagpie — COMMENTED** [review]
Im not sure why I don't have a approve button, but lgtm

**Pull request merged**